src/postgraphile/postgraphile.ts

```diff
@@ -1,188 +1,18 @@
 import { Pool, PoolConfig } from 'pg'
```

Comment:
only changes below are changing where types are imported from, and migrating these types to a common interfaces file